daonode/src/main/java/bisq/daoNode/DaoNode.java

```diff
@@ -17,45 +17,184 @@
 
 package bisq.daoNode;
 
-import bisq.core.app.misc.AppSetup;
+
+import bisq.core.app.TorSetup;
 import bisq.core.app.misc.AppSetupWithP2PAndDAO;
+import bisq.core.app.misc.ExecutableForAppWithP2p;
+import bisq.core.app.misc.ModuleForAppWithP2p;
 import bisq.core.dao.state.DaoStateService;
-import bisq.core.network.p2p.inventory.GetInventoryRequestHandler;
+import bisq.core.dao.state.DaoStateSnapshotService;
+import bisq.core.user.Cookie;
+import bisq.core.user.CookieKey;
 import bisq.core.user.Preferences;
+import bisq.core.user.User;
+
+import bisq.network.p2p.P2PService;
+import bisq.network.p2p.P2PServiceListener;
+import bisq.network.p2p.peers.PeerManager;
+
+import bisq.common.Timer;
+import bisq.common.UserThread;
+import bisq.common.app.AppModule;
+import bisq.common.app.Version;
+import bisq.common.config.BaseCurrencyNetwork;
+import bisq.common.config.Config;
+import bisq.common.handlers.ResultHandler;
 
-import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.name.Names;
 
 import lombok.Getter;
-import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
+//todo not sure if the restart handling from seed nodes is required
 
 @Slf4j
-public class DaoNode {
-    @Setter
-    private Injector injector;
-    private GetInventoryRequestHandler getInventoryRequestHandler;
+public class DaoNode extends ExecutableForAppWithP2p {
+    private static final long CHECK_CONNECTION_LOSS_SEC = 30;
+
+    private Timer checkConnectionLossTime;
     @Getter
     private DaoStateService daoStateService;
 
     public DaoNode() {
+        super("Bisq Dao Node", "bisq-dao-node", "bisq_dao_node", Version.VERSION);
```

Comment:
Is there some naming convention for the `appName` parameter value?  Is the convention snake_case now?

Reply:
No convention... Prefer to leave it as I used it in scripts already.